internal/cluster/refresh_status_preprocessor.go

```diff
@@ -185,5 +185,21 @@ func newConditions() []condition {
 			id: VipDhcpAllocationSet,
 			fn: isVipDhcpAllocationSet,
 		},
+		{
+			id: AllHostsPreparedSuccessfully,
+			fn: areAllHostsPreparedSuccessfully,
+		},
+		{
+			id: InsufficientHostExists,
+			fn: isInsufficientHostExists,
+		},
+		{
+			id: ClusterPreparationSucceeded,
+			fn: isClusterPreparationSucceeded,
+		},
+		{
+			id: ClusterPreparationFailed,
+			fn: isClusterPreparationFailed,
```

Comment:
Why does both validations are needed? Isn't `PreparationSucceeded` enough?

Reply:
No, because when there is no result the preparation didn't succeed, but it didn't fail